doc/devbox_setup.md

```diff
@@ -0,0 +1,115 @@
+# Prepare your development environment
+
+This document describes how to prepare your development environment to build and use the **Microsoft Azure IoT device client SDK for .NET**
+
+1.  [Prerequisites required to build the SDK](#min_setup)
+2.  [Optional prerequisites required to test Xamarin, Windows IoT and .NET Micro Framework](#advanced)
+3.  [Test prerequisites](#testprereq)
+
+<a name="min_setup"/>
+
+## Prerequisites required to build the SDK
+
+The following is the minimum required to build and produce IoT SDK binaries. 
```

Comment:
You would also need VS right?

Reply:
No. Since you've asked, that's not clear so I'll add something.